Modules/SlashCommands.cs

```diff
@@ -235,6 +236,14 @@ public class SlashCommands : ApplicationCommandModule
                 await ctx.RespondAsync($"{Program.cfgjson.Emoji.Error} The source user has no warnings to transfer.", Warnings.GenerateWarningsEmbed(sourceUser));
                 return;
             }
+            else if (merge)
+            {
+                await Program.db.KeyDeleteAsync(sourceUser.Id.ToString());
```

Comment:
Better to delete the source users warnings *after* the operation is complete, to avoid irreversibly destroying any data in the event that the rest of the code fails.

Reply:
I moved `KeyDeleteAsync` to below the `foreach` loop now.